Dockerfile.test.rhel7

```
@@ -0,0 +1,63 @@
+FROM registry.access.redhat.com/ubi7/ubi
+
+ENV INSTALL_DIR=/opt/conjur-api-python3
+
+# Copy below repo file to provide missing rpms
+COPY ./test/RPM-GPG-KEY-CentOS-7 /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+COPY ./test/CentOS-SCLo-scl-rh.repo /etc/yum.repos.d/CentOS-SCLo-scl-rh.repo
+COPY ./test/CentOS-7-Linux-AppStream.repo /etc/yum.repos.d/CentOS-Linux-AppStream.repo
+RUN gpg --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+
+# enable repos
+RUN yum-config-manager --enable ubi-server-rhscl-7-rpms
+RUN yum-config-manager --enable ubi-7-server-optional-rpms
+
+RUN yum --disableplugin=subscription-manager \
+                                install -y  bash \
+                                            wget \
+                                            binutils \
+                                            yum-utils \
+                                            gcc gcc-c++ make \
+                                            curl \
+                                            jq \
+                                            libffi-devel \
+                                            gnome-keyring \
+                                            procps \
+                                            zlib-devel \
+                                            wget \
+                                            openssl-devel \
+                                            bzip2-devel \
+                                            dbus-x11 \
+                                            && yum --disableplugin=subscription-manager clean all
```

I would change the indentation to this:
```suggestion
RUN yum \
    --disableplugin=subscription-manager \
    install -y \
      bash \
      wget \
      binutils \
      yum-utils \
      gcc gcc-c++ make \
      curl \
      jq \
      libffi-devel \
      gnome-keyring \
      procps \
      zlib-devel \
      wget \
      openssl-devel \
      bzip2-devel \
      dbus-x11 \
    && yum --disableplugin=subscription-manager clean all \
```

---

if you think it's better please do this for the rest of the file